src/user-feedback.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Options object for `Bugfender.getUserFeedback` with customised modal strings.
+ */
+export interface UserFeedbackOptions {
+  /** Default: `Feedback`. */
+  title?: string;
+  /** Default `Please insert your feedback here and click send`. */
+  hint?: string;
+  /** Default: `Subject…`. */
+  subjectPlaceholder?: string;
+  /** Default: `Your feedback…`. */
+  feedbackPlaceholder?: string;
+  /** Default: `Send`. */
+  submitLabel?: string;
+  /** Default: `Close`. This attribute is only used on iOS */
+  closeLabel?: string;
+}
+
+export class DefaultUserFeedbackOptions implements Required<UserFeedbackOptions> {
+  constructor() {
+    this.title = 'Feedback';
+    this.hint = 'Please insert your feedback here and click send';
+    this.subjectPlaceholder = 'Subject…';
+    this.feedbackPlaceholder = 'Your feedback…';
+    this.submitLabel = 'Send';
+    this.closeLabel = 'Close';
+  }
+
+  feedbackPlaceholder: string;
+  hint: string;
+  subjectPlaceholder: string;
+  submitLabel: string;
+  title: string;
+  closeLabel: string;
+}
```

Comment:
You can skip the duplication. Also, since it's a class it's better if we specify the props visibility (even if it defaults to `public`). Just to keep it in line with what we try to do in MJ.

```suggestion
export class DefaultUserFeedbackOptions implements Required<UserFeedbackOptions> {
  public title = 'Feedback';
  public hint = 'Please insert your feedback here and click send';
  public subjectPlaceholder = 'Subject…';
  public feedbackPlaceholder = 'Your feedback…';
  public submitLabel = 'Send';
  public closeLabel = 'Close';
}
```